apps/ledger-live-mobile/src/screens/PostOnboarding/PostOnboardingDebugScreen.tsx

```diff
@@ -1,21 +1,29 @@
 import { DeviceModelId } from "@ledgerhq/devices/index";
 import { Flex } from "@ledgerhq/native-ui";
 import React, { useCallback } from "react";
+import { useNavigation } from "@react-navigation/native";
 import { useStartPostOnboardingCallback } from "@ledgerhq/live-common/postOnboarding/hooks/index";
 import PostOnboardingEntryPointCard from "../../components/PostOnboarding/PostOnboardingEntryPointCard";
 import SettingsRow from "../../components/SettingsRow";
 import { useNavigateToPostOnboardingHubCallback } from "../../logic/postOnboarding/useNavigateToPostOnboardingHubCallback";
+import { NavigatorName } from "../../const";
 
 export default () => {
+  const navigation = useNavigation();
   const startPostOnboarding = useStartPostOnboardingCallback();
 
   const handleInitFTSMock = useCallback(
     () => startPostOnboarding(DeviceModelId.nanoFTS, true),
     [startPostOnboarding],
   );
   const handleInitFTS = useCallback(
-    () => startPostOnboarding(DeviceModelId.nanoFTS),
-    [startPostOnboarding],
+    () =>
```

Comment:
If you do the modification here, do it in other `handleInit...` callbacks no? (there is probably a way to refactor it so you don't have to repeat the same code btw)